src/System.Globalization.Calendars/tests/ISOWeek/ISOWeekTests.netcoreapp.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Globalization
+{
+    public static class ISOWeekTests
+    {
+        // The following 71 years in a 400-year cycle have 53 weeks (371 days).
+        // Years not listed have 52 weeks (364 days). Add 2000 for current years.
+        private static readonly HashSet<int> s_yearsWith53Weeks = new HashSet<int>
```

Comment:
Thinking I can just move this into a local now...

Reply:
if you think we'll not add more test cases in the future which using it

---
In reply to: [196166639](https://github.com/dotnet/corefx/pull/30386#discussion_r196166639) [](ancestors = 196166639)

Reply:
Meh, I guess it doesn't matter where the data is. Might as just keep it as a field like this.